### ISSUE TITLE
x-axis values move on container resize, introduced with v4.0.x

### ISSUE THREAD

**Issue by yphoenix** [fusioncharts/fusioncharts-dist#68]
A Bar2D chart displays just fine, but if you have the width set to 100% and you resize the container the x-axis values jump up onto the bottom border of the chart box. This didn't happen in earlier versions.

Here is a demo, run, displays OK, resize the window, and you'll see the bug.

https://jsfiddle.net/dtupqse3/

**Comment by yphoenix** [comment]
Can confirm NOT FIXED in 4.0.2

**Comment by yphoenix** [comment]
Can confirm NOT FIXED in 4.1.0

**Comment by Mettbrot** [comment]
We are also affected by this. Will this be addressed soon?

**Comment by yphoenix** [comment]
4.1.2 Seems worse, not that anybody is actually listening here!

**Comment by Mettbrot** [comment]
We contacted support, they said 

> This issue is raised internally and prioritised for next release.
> 
> However, the ETA for next release is yet to be confirmed and internal product team is working on it.

I think we will be moving away from fusioncharts at some point…

**Comment by yphoenix** [comment]
Yep, we're stuck on 3.21 because of bugs introduced on all versions since. Bugs we reported get fix, but new bugs get introduced so we can't move onwards One page of our app is stuck on 3.17 because of a bug that wasn't fixed until 4.0, but we can't use 4.0.

Since IDERA brought them I get a sense of more effort to raise revenue than there is to fix bugs. Just noticed some AI bot on their web-site. Stop putting effort into that and fix the bugs that keep people paying your annual license fees.

Long-term if things don't get fixed, we'll probably migrate elsewhere too.

**Comment by yphoenix** [comment]
Appears to be fixed in 4.1.3-beta.1
